### PR TITLE
Updating minimum PHP version to 5.6

### PR DESCRIPTION
**Pull request by rawkode**
I am targetting this branch, because it's a PHP version change / potential BC break

Refs # https://github.com/sonata-project/dev-kit/issues/208

## Changelog

``` markdown
### Removed
- Support for PHP versions 5.3 - 5.5
```

## Subject

Updating minimum PHP version to 5.6, as per https://github.com/sonata-project/dev-kit/issues/208

### CODE REVIEW

**Comment by rawkode** [comment]
Don't think this is likely to work with Symfony 2.3

**Comment by greg0ire** [comment]
We're supposed to drop sf 2.3 => 2.7 on master.

**Comment by rawkode** [comment]
All I've done is remove code, I'm not sure why coveralls is having a moan. Hopefully still mergable!

@greg0ire

**Comment by greg0ire** [comment]
> I'm not sure why coveralls is having a moan.

Probably because the code you removed was well-covered ;) But this is a non-blocking issue. Can't review that right now, but I will have a look at it, thanks!

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +10,6 @@ branches:
 language: php
 
 php:
-  - '5.3'
-  - '5.4'
-  - '5.5'
```
Comment (greg0ire) [inline .travis.yml]: This is supposed to be done via dev-kit

File: composer.json [hunk 2]
```diff
@@ -17,22 +17,22 @@
...
+        "symfony/finder": "^2.7 || ^3.0",
+        "symfony/form": "^2.7 || ^3.0",
+        "symfony/http-foundation": "^2.7 || ^3.0",
+        "symfony/http-kernel": "^2.7 || ^3.0",
+        "symfony/options-resolver": "^2.7 || ^3.0",
+        "symfony/routing": "^2.7 || ^3.0",
+        "symfony/security":"^2.7 || ^3.0",
+        "symfony/templating":"^2.7 || ^3.0",
+        "symfony/translation": "^2.7 || ^3.0",
+        "symfony/validator": "^2.7 || ^3.0",
```
Comment (greg0ire) [inline composer.json]: Should be 2.8 here I think.

**Pull request closed without merging**